fixtures/nesting/package.json

```diff
@@ -0,0 +1,42 @@
+{
```

Comment:
Should we either add the package-lock.json files are gitignore them?

Reply:
They generally shouldn't be ignored. I'll commit them if we move this to another repo.